datastream/consts.go

```diff
@@ -0,0 +1,41 @@
+package datastream
+
+import "time"
+
+const defaultBaseURL = "api.schematic.com"
```

Comment:
This can be part of the next PR, but we probably need to allow setting the scheme as well, since we'll want to be able to test this locally with ws:// and live with wss://